packages/dropdowns/demo/stories/DropdownFieldStory.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { HTMLAttributes } from 'react';
+import { Story } from '@storybook/react';
+import {
+  Field,
+  Hint,
+  IDropdownProps,
+  IMenuProps,
+  IMessageProps,
+  Label,
+  Message
+} from '@zendeskgarden/react-dropdowns';
+import { DropdownStory } from './DropdownStory';
+import { ITEM } from './types';
+import { IMenuItemProps } from './MenuStory';
+
+interface IArgs extends HTMLAttributes<HTMLDivElement> {
+  dropdownProps: IDropdownProps;
+  label?: string;
+  isLabelRegular?: boolean;
+  isLabelHidden?: boolean;
+  hasHint?: boolean;
+  hint?: string;
+  hasMessage?: boolean;
+  message?: string;
+  validation?: IMessageProps['validation'];
+  menuProps?: IMenuProps;
+  items: ITEM[];
+  itemProps?: IMenuItemProps;
+}
+
+export const DropdownFieldStory: Story<IArgs> = ({
+  dropdownProps,
+  label = 'Label',
+  isLabelRegular,
+  isLabelHidden,
+  hasHint = true,
+  hint = 'Hint',
+  hasMessage = true,
+  message = 'Message',
+  validation,
```

Comment:
I couldn't tell where this was defined [in the story](https://github.com/zendeskgarden/react-components/blob/82561ad10fb06cedacae3d26b3426f764d8d741c/packages/dropdowns/demo/combobox.stories.mdx). Can you point me to where it can be found?

Reply:
In dropdowns (and fields) `validation` is a concept that spans both the input and associated `Message` component. In the case you referenced, it applies styling to both the `Combobox` and associated `Field` > `Message` component. The same control can be seen with dropdowns `Autocomplete`, `Multiselect`, `Select` and similar "Forms" components.

We don't have solid design evidence that validation should apply (as it currently does) disparate from input & message. But the current implementation allows for cases we don't know about.

Reply:
p.s. The MDX story does not (should not) need to expose props as args which are inherent in the component identified in `<Meta>`. Which, in this case, is `Combobox` and includes `validation`, `focusInset`, `isBare`, `isCompact`, etc.